bftengine/src/bftengine/messages/ClientReplyMsg.cpp

```diff
@@ -59,6 +59,11 @@ void ClientReplyMsg::setReplicaSpecificInfoLength(uint32_t length) {
   b()->replicaSpecificInfoLength = length;
 }
 
+void ClientReplyMsg::setResult(uint32_t result) {
+  ConcordAssert(result <= maxReplyLength());
+  b()->result = result;
+}
+
```

Comment:
Not needed anymore

Reply:
Its needed to set the result in allocateNewReplyMsgAndWriteToStorage